<launch>
	<!-- device driver /-->
    <include file="$(find application)/launch/devices.launch"/>

 	<!-- uav controller launch file -->
    <node name="uav_controller_server_node" pkg="uav_controller" type="uav_controller_server" output="screen"/>

	<!-- rrt planner server /-->
    <node name="rrt_planner_server" pkg="rrt_path_planner" type="rrt_planner_node" output="screen"/>

	<!-- linear planner server /-->
	<node name="linear_planner_server" pkg="linear_path_planner" type="linear_planner_node" output="screen"/>

    <!-- octomap mapping /-->
	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server_node" args="$(env HOME)/Documents/optimizedMap.bt" output="screen">
		<param name="frame_id" value="/map"/>
		<param name="resolution" value="0.09" />
		<param name="latch" value="true"/>
		<param name="pointcloud_max_z" value="3.5" />
		<param name="pointcloud_min_z" value="0.2" />
		<param name="sensor_model/max_range" value="4.0" /> 
		<param name="sensor_model/hit" value="0.64" /> 
		<param name="sensor_model/max" value="0.97" /> 
		<param name="filter_ground" value="false" /> 
        <param name="occupancy_min_z" value="0.6" />
        <param name="occupancy_max_z" value="1.0" />
	</node>
	
	<!-- exploration node -->
	<node name="localization_node" pkg="application" type="localization_node" output="screen"/>
</launch>